service/controller/clusterapi/cluster.go

```diff
@@ -3,7 +3,7 @@ package clusterapi
 import (
 	"net"
 
-	"github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
+	clusterv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/cluster/v1alpha1"
```

Comment:
Why did this change though?

Reply:
Because now cluster controller is using Cluster CRD instead of AWSConfig CRD.